guides/doc-Administering_Project/master.adoc

```diff
@@ -60,6 +60,8 @@ include::common/assembly_logging-and-reporting-problems.adoc[leveloffset=+1]
 
 include::common/assembly_monitoring-resources.adoc[leveloffset=+1]
 
+include::common/assembly_limiting-resources.adoc[leveloffset=+1]
```

Comment:
The plugin is pretty cool, _all_ downstream products should deliver it :) If that's not yet the case, please let me now and I can limit the visibility here. :upside_down_face: